tubular/scripts/retrieve_latest_base_ami.py

```diff
@@ -73,15 +73,19 @@ def retrieve_latest_base_ami(environment, deployment, play, override, ubuntu_ver
         else:
             url = ""
             click.secho('Ubuntu version requested: {}'.format(ubuntu_version), fg='green')
-            if ubuntu_version == "20.04":
+            # NOTE: The next LTS, noble, isn't present. Not sure why.
+            if ubuntu_version == "22.04":
+                url = "https://cloud-images.ubuntu.com/query/jammy/server/released.current.txt"
+                click.secho('Jammy.\n: {}'.format(url), fg='green')
+            elif ubuntu_version == "20.04":
```

Comment:
Author's note: I would prefer to reverse the order of these conditionals so we can just add a new `elif`, but I'm trying to keep the diff small.